packages/vscode-extension/src/common/utils.ts

```diff
@@ -27,3 +27,25 @@ export function throttle<T extends (...args: any[]) => any>(
     recentArgs = args;
   } as T;
 }
+
+export function throttleWithTrailing<T extends (...args: any[]) => any>(
```

Comment:
how is this different than "throttle"? Maybe add some comment here? Also from reading the code it appears like you can actually use the existing method, no?